examples/collector-agent-host-network-disabled/README.md

```diff
@@ -0,0 +1,8 @@
+# Example of chart configuration
+
+## Deploy the Splunk Collector in agent mode and disable using the host network
+
+- This configuration will install the collector as an agent deployment only.
+- Disabling this value will affect monitoring of some control plane components.
```

Comment:
What is `this value` in this context?

Reply:
Some metrics will be missing, but I don't think we know which ones will be missing. I'll see if I can use the tests in this PR to determine which metrics will be missing and add them here.

Reply:
Paulo is asking what "this value" is literally as in - which toggle is being set to false?

Reply:
to make this more obvious to me (hopefully I am representing well what Paulo is saying):
```suggestion
- Setting `agent.hostnetwork` to false will affect monitoring of some control plane components.
```